cfp.md

```diff
@@ -34,18 +34,61 @@ Contributions may overlap with any of the following areas:
 - Time-predictable hardware
 
 More background about this workshop and its motivation can be found [here]({% link about.md %}).
+
 ## Important Dates
-TBA
+- July 8, 2024: Paper submission deadline
+- Aug 5, 2024: Author notification for the first round of reviews
+- Aug 12, 2024: Camera-ready and video graphical abstract deadline
```

Comment:
What does this mean?

Reply:
I read on and now know what it means, but I think it would be better to specify this earlier. Maybe before the Important Dates, we should have a section that outlines the different tracks, and what authors should expect to deliver (i.e., graphical abstract for IEEE ESL). We can then go into the details of the logistics further down the page.

Reply:
I reorganized the content.

Reply:
This looks good!